SPECS/edk2/CVE-2023-45232.nopatch

```diff
@@ -0,0 +1,2 @@
+CVE already patch in CVE-2023-45230.patch 
```

Comment:
For the nopatch files, it is not preferable to drop them and either mention all fixed CVEs in the actually applied patch file (`CVE-2023-45230.patch` in this case) or file a dispute through Astrolabe saying the CVE was fixed in a different patch file.

Reply:
How would mentioning them work? The tooling wouldn't know we had patched the other CVEs right?

Reply:
That's the guidelines I got from the CVE specialists. I'm guessing they might be scanning the patch files. Or at least that's the only thing, that comes to my mind.

Reply:
After discussion with CVE team. We are ok to use nopatch method.